packages/ui/src/components/Button/style.less

```diff
@@ -322,3 +322,63 @@ a.condo-btn {
     @condo-button-border-width);
   }
 }
+
+.condo-dropdown-trigger.condo-btn-extended {
```

Comment:
Should it be in `Dropdown` / `Dropdown.button` styles?

Reply:
moved to `Dropdown.button ` styles